app/src/main/AndroidManifest.xml

```diff
@@ -4,24 +4,25 @@
     package="com.example.event_finder_capstone">
 
     <application
+        android:name=".ParseApplication"
         android:allowBackup="true"
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="Event Finder"
```

Comment:
🛠 We generally want to store strings in the resource file to support localization - https://developer.android.com/guide/topics/resources/localization.html